python/ql/test/query-tests/Functions/general/protocols.py

```diff
@@ -118,4 +118,8 @@ def __call__(self):
         raise StopIteration
 
     def __bool__(self):
-        raise TypeError
\ No newline at end of file
+        raise TypeError
+
+class BadBool(object):
+    def __bool__(self):
+        raise ZeroDivisionError()
```

Comment:
nitpick: could we move this to `IncorrectSpecialMethods`?

Reply:
Yes! Will do.